### PR TITLE
Release: 0.19.6

### PR DESCRIPTION
**Pull request by ahmedetefy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-
-- Semver says that major version `0` can include breaking changes at any time. Still, it is common practice to assume that only `0.x` releases (minor versions) can contain breaking changes while `0.x.y` releases (patch versions) are used for backwards-compatible changes (bugfixes and features). This project also follows that practice.
+We decided to use [semver](https://semver.org/) versioning from now on for the SDK. with two additions:
 
 - All undocumented APIs are considered internal. They are not part of this contract.
-
 - Certain features (e.g. integrations) may be explicitly called out as "experimental" or "unstable" in the documentation. They come with their own versioning policy described in the documentation.
 
-We recommend to pin your version requirements against `0.x.*` or `0.x.y`.
-Either one of the following is fine:
```
Comment (HazAT) [inline CHANGELOG.md:9]: I think those parts shouldn't change yet

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
 
-```
-sentry-sdk>=0.10.0,<0.11.0
-sentry-sdk==0.10.1
-```
+## 0.19.6
+
+The bump to 1.0 contains a breaking change.
```
Comment (HazAT) [inline CHANGELOG.md:12]: Remove this

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
 
-A major release `N` implies the previous release `N-1` will no longer receive updates. We generally do not backport bugfixes to older versions unless they are security relevant. However, feel free to ask for backports of specific commits on the bugtracker.
+- Fix for header extraction for AWS lambda/API extraction
+- Fix multiple **kwargs type hints # 967
+- Fix that corrects AWS lambda integration failure to detect the aws-lambda-ric 1.0 bootstrap #976
+- Fix AWSLambda integration: variable "timeout_thread" referenced before assignment #977
+- Use full git sha as release name #960
+- **BREAKING CHANGE**: The default environment is now production, not based on release
```
Comment (HazAT) [inline CHANGELOG.md:19]: And not sure about this

**Review by HazAT — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (ahmedetefy) [reply CHANGELOG.md:17]: What would you change about it?

Comment (HazAT) [reply CHANGELOG.md:30]: Like we shouldn't have the breaking change in there

Comment (ahmedetefy) [reply CHANGELOG.md:30]: Should I revert that commit?

Comment (untitaker) [reply CHANGELOG.md:30]: just do 0.20 if you're not happy about this. The previous changelog entry is also a breaking change in the same fashion (it changes the default release name)

**Pull request closed without merging**